Utilities/VSOBuildScripts/postBuildScript.cmd

```diff
@@ -0,0 +1,43 @@
+if "%1" == "local" goto testlocal
```

Comment:
This is consistent with https://github.com/Microsoft/xbox-live-api and the rest of the XBL github repos.